applications/GeoMechanicsApplication/custom_constitutive/tension_cutoff.cpp

```diff
@@ -20,15 +20,15 @@ namespace Kratos
 {
 TensionCutoff::TensionCutoff(double TensileStrength) : mTensileStrength{TensileStrength} {}
 
-double TensionCutoff::YieldFunctionValue(const Vector& rPrincipalStress) const
+double TensionCutoff::YieldFunctionValue(const Vector& rSigmaTau) const
 {
-    return rPrincipalStress(0) - mTensileStrength;
+    return rSigmaTau[0] + rSigmaTau[1] - mTensileStrength;
 }
 
-Vector TensionCutoff::DerivativeOfFlowFunction(const Vector& rPrincipalStress) const
+Vector TensionCutoff::DerivativeOfFlowFunction(const Vector& rSigmaTau) const
```

Comment:
Since we don't use this function parameter, let's "unname" it:
```suggestion
Vector TensionCutoff::DerivativeOfFlowFunction(const Vector&) const
```